dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -196,6 +196,154 @@
       }
      ]
     },
+    {
+     "type": "panel",
+     "name": "friday_institute_panel",
+     "elements": [
+      {
+       "type": "html",
+       "name": "question3",
+       "visible": false,
+       "html": "<p><span style=\"background-color: #ffff00;\"> &gt;&gt; FRIDAY-INSTITUTE-SPECIFIC&lt;&lt;</span></p>"
+      },
+      {
+       "type": "radiogroup",
+       "name": "friday_institute_which_tool",
+       "title": "Which video conferencing tool was primarily used for the synchronous (live meeting) portions of the workshop?",
+       "choices": [
+        {
+         "value": "hangouts",
+         "text": "Google Hangouts"
+        },
+        {
+         "value": "zoom",
+         "text": "Zoom"
+        },
+        {
+         "value": "teams",
+         "text": "Microsoft Teams"
+        },
+        {
+         "value": "goto_meeting",
+         "text": "GoToMeeting"
+        },
+        {
+         "value": "skype",
+         "text": "Skype"
+        },
+        {
+         "value": "dont_know",
+         "text": "Don't Know/Unsure"
+        }
+       ],
+       "hasOther": true,
+       "otherText": "Other"
+      },
+      {
+       "type": "matrix",
+       "name": "friday_institute_sync_tools",
+       "title": "Thinking about the video conferencing portions of the workshop, how much do you agree with the following?  ",
```

Comment:
remove extra spaces at the end of the sentence?

Reply:
fixed